gen/calc.py

```diff
@@ -387,7 +387,7 @@ def calculate_config_yaml(user_arguments):
 def calculate_mesos_isolation(enable_gpu_isolation):
     isolators = ('cgroups/cpu,cgroups/mem,disk/du,network/cni,filesystem/linux,'
                  'docker/runtime,docker/volume,volume/sandbox_path,posix/rlimits,'
-                 'namespaces/pid,com_mesosphere_MetricsIsolatorModule')
+                 'namespaces/pid,linux/capabilities,com_mesosphere_MetricsIsolatorModule')
```

Comment:
ok, I was thinking if we had a test that exerts the different isolators supported (which would fail with this, but will require an update as this new isolator is introduced).

It seems we don't have that, and it may not be a requirement for this PR change too.

So, the change lgtm.